java/example/src/main/java/io/myabcwallet/Signup.java

```diff
@@ -311,8 +311,8 @@ public static void main(String[] args) throws Exception {
     }
 
     /*
-     *  1.  :man_raising_hand: Getting Started > Secure Channel 참고 ([getting-started/guide/login/](secure-channel.md#__tabbed_1_2))
-     *  2.  :man_raising_hand: Getting Started > Secure Channel 참고 ([getting-started/guide/login/](secure-channel.md#__tabbed_1_2))
-     *  3.  :man_raising_hand: 사전에 발급받은 Client ID / Client Secret 이 필요합니다. Client ID 와 Client Secret 을 base64 로 인코딩 해야 합니다.
-     */
+    1.  :man_raising_hand: Getting Started > Secure Channel 참고 ([getting-started/guide/login/](secure-channel.md#__tabbed_1_2))
+    2.  :man_raising_hand: Getting Started > Secure Channel 참고 ([getting-started/guide/login/](secure-channel.md#__tabbed_1_2))
+    3.  :man_raising_hand: 사전에 발급받은 Client ID / Client Secret 이 필요합니다. Client ID 와 Client Secret 을 base64 로 인코딩 해야 합니다.
```

Comment:
```suggestion
    1.  :man_raising_hand: Getting Started > Secure Channel 참고 ([getting-started/guide/secureChannle/](secure-channel.md#__tabbed_1_4))
    2.  :man_raising_hand: Getting Started > Secure Channel 참고 ([getting-started/guide/login/](secure-channel.md#__tabbed_1_4))
    3.  :man_raising_hand: 사전에 발급받은 Client ID / Client Secret 이 필요합니다. Client ID 와 Client Secret 을 base64 로 인코딩 해야 합니다.
```

설명에 맞는 페이지로 갈 수 있도록 수정이 필요할 것 같습니다.
1- ts
2-python
3-go
4-java